### DISCUSSION TITLE
How do I add an attachment to a reply in Thunderbird Conversations

### DISCUSSION THREAD

**Discussion by thermopyle** [protz/thunderbird-conversations#2025] [Q&A] [answered]
I have replied to an email in the Conversations view and need to add an attachment to the reply, but I totally fail to understand how I do this. I cannot find any indication anywhere of how this is done. It is possible, I guess?

**Comment by Standard8** [comment, marked as answer]
It is not currently possible to add attachments in the quick reply view. You can use the normal compose window for attachments (via the reply in the three-dots menu at the top-right of the message, or via Thunderbird's normal reply functions).

**Comment by thermopyle** [comment]
Great, thanks for the quick reply!
